_posts/2018-09-05-Data-Continuity-Service-for-DynamoDB.md

```diff
@@ -0,0 +1,570 @@
+---
+layout: post
+title: Data Continuity Service for DynamoDB
+banner: /assets/posts/2018-09-05-Data-Continuity-Service-for-DynamoDB/solution-overview.png)
+author: tvb
+---
+
+This is the first post of three where I am going to explain how to build and configure a Data Continuity Service (DCS) for Amazon DynamoDB by using Amazon DynamoDB Streams, AWS Lambda and the Amazon S3 services.
+
+Why Data Continuity Service?
+-----
+Now, you might ask yourself why you would need such thing as Data Continuity Service (DCS) for DynamoDB because AWS launched the [backup and restore](https://aws.amazon.com/about-aws/whats-new/2017/11/aws-launches-amazon-dynamodb-backup-and-restore/) functionality for DynamoDB in November 2017.
+Well, while this is true and it works really great there are some shortcomings to the backup and restore functionality. Which I will explain now.
+
+First, don't get me wrong! You should be using the backup and restore functionality integrated with Amazon DynamoDB always as it is really great at quickly making backup and restore your content. However it also means these backups are hosted in the same region and within the same account as your DynamoDB data is located.
```

Comment:
>However it also means these backups are hosted in the same region and within the same account as your DynamoDB data is located.

There is something grammatically wrong here. I'd change it to:

>However it also means these backups are hosted in the same region and within the same account as your source DynamoDB data.

Reply:
Thanks